### DISCUSSION TITLE
Full Width Theme Option

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#948] [Ideas]
Would you be open to a PR that adds a config value that allows the user to always hide the sidebar, instead of only on smaller screens?

**Comment by danharrin** [comment]
Hmm, yeah why not :)

**Comment by intrepidws** [comment]
OK, I'll work on that. I was also thinking about allowing people to specify the sidebar width in the config file.

Out of curiosity, did you have plans to allow people to easily make changes to the layout files in the future?

**Reply by danharrin** [reply]
My only plans would be to further extract everything into smaller Blade components that make overriding easier

**Reply by intrepidws** [reply]
Is there documentation that explains how overriding is possible?

**Reply by danharrin** [reply]
You can run `php artisan vendor:publish --tag=filament-views`, and delete the files you don't want to override

**Reply by SebastiaanKloos** [reply]
Ohh. Wow. Didn't know that was possible. Love it❤️
